godotengine/godot
# Implement Set inside Variant

closes https://github.com/godotengine/godot-proposals/issues/867

- [x] GDScript
     - [x] Non-typed test
     - [ ] Typed test

TODO:
 - [x] Modify the VariantParser to no longer use ERR_PRINTER_ON_FIRE to signal from the dictionary parser that it's actually a set.
 - [x] Modify for it to support sets with one element.
 - [x] Make the unit tests stop failing even if the output is correct.
 - [x] Implement typed sets
 - [ ] Fix issues that make the typed sets unit test fail.
 
 TODO AFTER MERGE:
 - Mono
 - Make sets exportable.
 - Add set to godot-cpp
